### DISCUSSION TITLE
Store more geo-locator related data

### DISCUSSION THREAD

**Discussion by ApaDoctor** [oscarmcm/django-places#42] [General]
### DESCRIPTION
Hey, you have very nice plugin. But very important thing is missing. 
Pieces of location in the modelfield attrs(as coords) - city, street, house number etc.
And second important thing - requirements to the accuracy of address. 
For example address accuracy level - `house number' or just 'city'

Thank you!

**Comment by oscarmcm** [comment]
Yeah, probably we can save more data coming from the geo-locator response, but I think it's too much for just a simple CharField, so maybe we should move away from the CharField and use a JSONField for store more data coming from the response.

 If its more people interested in this feature, I could add this to package. 

Leaving this issue open for more discussion.
